uint8 status
Header header
float64 x
float64 y
float64 xmin
float64 xmax
float64 ymin
float64 ymax
float64 highLevTime
float64 term_xmin
float64 term_xmax
float64 term_ymin
float64 term_ymax